client/packages/common/src/types/schema.ts

```diff
@@ -2858,7 +2863,7 @@ export type StockLineSortInput = {
 export type StockLinesReducedBelowZero = UpdateStocktakeErrorInterface & {
   __typename: 'StockLinesReducedBelowZero';
   description: Scalars['String'];
-  stockLines: StockLineConnector;
+  errors: Array<StockLineReducedBelowZero>;
```

Comment:
`StockLinesReducedBelowZero` to return array of `StockLineReducedBelowZero`. If error occurs during finalisation, it is displayed in main table and when we look at batch table, error should also be displayed (just by opening it, if error was already know when we tried to finalise stocktake), since batch table needs to display errors based on StockLine (since it shows draft lines, and they may not be inserted yet), it's good to use one common error for each stocktake line, thus collection of `StockLineReducedBelowZero` errors that can be mapped to each stocktake line.